apps/base/admin.py

```diff
@@ -1,5 +1,11 @@
 from django.contrib import admin
 
+from import_export.admin import ImportExportMixin
+
 
 class TimeStampedAdmin(admin.ModelAdmin):
     exclude = ('created_at', 'modified_at',)
+
+
+class ImportExportTimeStampedAdmin(ImportExportMixin, admin.ModelAdmin):
+    exclude = ('created_at', 'modified_at',)
```

Comment:
I think the challenge host or EvalAI Admin would need these fields to analyze the data based on timestamp, so let's not remove these fields.